ui/component/settingNotifications/view.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import Card from 'component/common/card';
+import { FormField } from 'component/common/form';
+import SettingsRow from 'component/settingsRow';
+import ChannelSelector from 'component/channelSelector';
+
+type Props = {
+  // --- select ---
+  notificationSettings: ?NotificationSettings,
+  // --- perform ---
+  doFetchNotificationSettings: () => void,
+  doToggleNotificationsDisabled: () => void,
+  doToggleFollowerNotifications: () => void,
+  doToggleFollowedNotifications: () => void,
+};
+
+export default function SettingNotifications(props: Props) {
+  const {
+    notificationSettings,
+    doFetchNotificationSettings,
+    doToggleNotificationsDisabled,
+    doToggleFollowerNotifications,
+    doToggleFollowedNotifications,
+  } = props;
+
+  const { disabled, mention } = notificationSettings || {};
+  const { all: notificationsDisabled } = disabled || {};
+  const { from_followed: followedMentionsEnabled, from_followers: followerMentionsEnabled } = mention || {};
+
+  React.useEffect(() => {
+    if (doFetchNotificationSettings && !notificationSettings) doFetchNotificationSettings();
+  }, [doFetchNotificationSettings, notificationSettings]);
+
+  return (
+    <>
+      <ChannelSelector hideAnon />
+
+      <Card
+        isBodyList
+        body={
+          <>
+            <SettingsRow title={__('Disable All')}>
+              <FormField
+                type="checkbox"
+                name="notifications_disable"
+                checked={notificationsDisabled}
+                onChange={doToggleNotificationsDisabled}
```

Comment:
Tried to toggle this but nothing happened.  Noticed `notification_settings/set` is returning a `405` for me.
Do I need to set up something first?

Reply:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/64950861/159848086-369d3e52-674c-4cfe-a002-831d86ed7399.png">

<img width="480" alt="image" src="https://user-images.githubusercontent.com/64950861/159847927-911ed226-b1ff-463c-8e82-5a08b9158f50.png">

Reply:
Oh, I just noticed this in the description.  Sorry.
> _(something wrong with the set API right now)_